spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  describe 'date formatting constants' do
+    context 'FULL_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::FULL_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+
+    context 'MONTH_YEAR_DATE_FORMAT_STRING' do
+      it 'exists' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_present
+      end
+
+      it 'is a String' do
+        expect(ApplicationHelper::MONTH_YEAR_DATE_FORMAT_STRING).to be_a_kind_of String
+      end
+    end
+  end
+
+  context ':format_full_date' do
+    it 'formats a date' do
+      expect_any_instance_of(Date).to receive(:strftime).
+                                  with(ApplicationHelper::FULL_DATE_FORMAT_STRING).
+                                  and_return '45 März 1776'
+      expect(format_full_date(Date.current)).to eq '45 März 1776'
+    end
+
+    it "doesn't raise exception on nil" do
```

Comment:
Echoing my earlier comment: the method doesn't raise exceptions no matter what object type it receives, which makes this test somewhat disingenuous.